web-24wi/assignments/nathanMcL/StudentOriginatedSoftware/week2/Calculator_app/calculator.js

```diff
@@ -0,0 +1,103 @@
+// Calculator state variables
+let runningTotal = 0;
+let buffer = "0";
```

Comment:
why keep track of the buffer and running total in separate variables?

Reply:
The reason to keep track of the buffer and the running total variables is because the buffer holds the current input from the user, it is a storage for the numbers and operators the user enters before they press enter. 
The runningTotal variable is used to keep track of the cumulative results from the operations performed